libsplinter/src/biome/rest_api/actix/mod.rs

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#[cfg(feature = "biome-key-management")]
+#[cfg(any(feature = "biome-key-management", feature = "biome-credentials"))]
```

Comment:
Should this be `all`?  Based on your commit message it sounds like both are required, not either.

Reply:
No, it's either. The authorize module is being used separately by both the key management and credentials routes.